### PR TITLE
Molpro properties

### PR DESCRIPTION
**Pull request by sjrl**
1. Added `calcinfo_nalpha`, `calcinfo_nbeta`, and `calcinfo_nbasis`
2. Renamed HF to RHF and KS to RKS to be clear that a restricted reference is being used
3. Added examples for unrestricted HF and KS (i.e. UHF and UKS)

## Question
- [ ] To request a unrestricted SCF I have added the keyword `reference` that can have value `unrestricted`. https://github.com/MolSSI/QCEngineRecords/compare/master...sjrl:molpro_properties?expand=1#diff-395fa842e2cd9f1029737b500c159ed4R13
I'm not sure this is the best choice, would it be better to follow what Psi4 does with the reference keyword? (http://www.psicode.org/psi4manual/1.2/autodoc_glossary_options_c.html#term-reference-scf)

### CODE REVIEW

**Comment by dgasmith** [comment]
We sort of fudge it [here](https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/psi4.py#L83). It is a reasonable assumption I think and makes high throughput much easier.

File: qcenginerecords/molpro/ccsd_prt_pr_water_gradient/output.json [hunk 1]
```diff
@@ -49,7 +49,10 @@
     "scf_total_energy": -74.9646625166939,
     "ccsd_prt_pr_correlation_energy": -0.0550552030560323,
     "ccsd_prt_pr_total_energy": -75.0197177197499,
-    "ccsd_prt_pr_dipole_moment": [0.0, 0.0, 0.606516899290546]
+    "ccsd_prt_pr_dipole_moment": [0.0, 0.0, 0.606516899290546],
+    "calcinfo_nalpha": 5,
+    "calcinfo_nbeta": 5,
+    "calcinfo_nbasis": 7
```
Comment (dgasmith) [inline qcenginerecords/molpro/ccsd_prt_pr_water_gradient/output.json:55]: Cool, great to add this to the parsing!

**Review by dgasmith — APPROVED** [review]
Overall LGTM!

**Comment by sjrl** [comment]
Thanks! Okay I'll switch over to allowing people to specify `uhf` and `uks` for the reference keyword then? Or did you mean my current set up with the value `unrestricted` was fine to keep?

File: qcenginerecords/molpro/ccsd_prt_pr_water_gradient/output.json (hunk 1 above)
Comment (sjrl) [reply qcenginerecords/molpro/ccsd_prt_pr_water_gradient/output.json:55]: Yes, unfortunately Molpro doesn't write out the nalpha and nbeta explicitly but I think I've figured out a workaround that still only needs the xml and the assumption that `nalpha > nbeta` which I believe is always true in Molpro.

Comment (dgasmith) [reply qcenginerecords/molpro/ccsd_prt_pr_water_gradient/output.json:55]: That is the assumption in most programs that I have worked with. It doesn't matter and might as well pick one.

**Comment by dgasmith** [comment]
I think your current version is fine!

**Pull request merged**